OSDOCS-16949#CQA work Stor5 - CSI Vol Populators (#114642)
openshift/openshift-docs: modules/persistent-storage-csi-vol-populator-procedure-top-level.adoc

Reviewer: This module will serve better as a concept that acts as a "parent" of the two procedure modules nested under it. As it is currently written, it's a bit confusing because it looks like a procedure, but the "real" procedures are in `modules/persistent-storage-csi-vol-populator-procedure.adoc` and `modules/persistent-storage-csi-vol-populator-procedure-admin.adoc`.

I provided some suggested revisions throughout the rest of this file for how you might change it to a concept. It will essentially be a module with only a short description, which Shauna mentioned is acceptable earlier in today's OCP core docs team meeting.

// Module included in the following assemblies:
//
// * storage/container_storage_interface/persistent-storage-csi-vol-populators.adoc

> :_mod-docs-content-type: PROCEDURE

Reviewer: ```suggestion
:_mod-docs-content-type: CONCEPT
```

[id="persistent-storage-csi-vol-populator-procedure-top-level_{context}"]

> = Creating volume populators

Reviewer: ```suggestion
= Volume populator creation
```
Changing the title to be a noun phrase, per the mod docs guidelines.

[role="_abstract"]

> To implement custom volume prepopulation behavior, create a volume populator by defining a custom resource definition and then use it to create prepopulated volumes.

Reviewer: ```suggestion
To implement custom volume prepopulation behavior, create a volume populator by defining a custom resource definition (CRD) and then using it to create prepopulated volumes.
```

> .Procedure

Reviewer: ```suggestion

```

> . Create a custom resource definition (CRD) for volume populators.

Reviewer: ```suggestion

```

> . Create a prepopulated volume using a volume populator.

Reviewer: ```suggestion

```
